azerothcore/azerothcore-wotlk
# fix(DB/Quest): Disabled quest A little luck (6606)

As the last sources ive found seems to prove that thgis quest was removed at some point (on classic wow is not showing, and in some wotlk addons seems to agree on that) ive changed the scope of this issue.

## Changes Proposed:
-  As the sources indicates that the quest was removed at some point on classic this quest should be disabled.

## Issues Addressed:

- Closes  https://github.com/azerothcore/azerothcore-wotlk/issues/9334
- Closes https://github.com/chromiecraft/chromiecraft/issues/2399

## SOURCE:

https://user-images.githubusercontent.com/87535580/145009847-3c56a5e3-7112-4a72-b375-8745991b15ea.png


## Tests Performed:

- .go c id 11755 and see if the quest A Little Luck (6606) is there.
- Apply fix
- Check now. The quest should not be there


## How to Test the Changes:


1.  .go c id 11755 and see if the quest A Little Luck (6606) is there.
2. Apply fix
3. Check now. The quest should not be there


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
